### ISSUE TITLE
How to login to a service through Google

### ISSUE THREAD

**Issue by fedemp** [stoically/temporary-containers#159]
This is not a bug but a call for help after reading the wiki.

- Temporary Containers Version: 0.90
- Firefox Version: 60.0.1
- Operating System + Version: Archlinux
- Non-default Options/Preferences in Temporary Containers:
Automatic mode.
All options in "Global isolation" set to "Subdomains wont get isolated".
Per domain: Added `*.mycompany.com`, "Always open...": disabled. Navigation and all clicks in "never". Excluded Target Domains set to `*.google.com`.
- Other installed Add-ons + Version:
Cookie AutoDelete 2.2.0
Decentraleyes 2.0.4
Facebook Container 1.3.1
Firefox Multi-Account Containers 6.0.0
HTTPS Everywhere 2017.6.1.1337
Neat URL 4.1.5
Temporary Containers 0.90
Vimium 1.63.3
Firefox Pioneer 1.0
uBlock Origin 1.16.8

### Actual behavior
When it visit `https://mycompany.com/login`, it is opened in a permanent container I have already set. In this page I have to click a "sign in with google" button. The sign in form is opened in a new temporary container, so the original tab does not have access to my Google credentials.

### Expected behavior
Google sign in form is opened in the same container as the login page.

So, TLDR: How can I configure a domain so google login is opened in the same container as said domain.

**Comment by fedemp** [comment]
Note: I have tried disabling and enabling the "multi account container" extension and now it seems to work. The Google login page is opened in the same tab and same container and the company login page.

**Comment by stoically** [comment]
Glad you figured it out. The Isolation feature needs some sort of logging tool so you can see when domains are isolated and actions to unbreak them. You can follow progress here https://github.com/stoically/temporary-containers/issues/104
